cms/templates/container.html

```diff
@@ -144,7 +144,7 @@ <h4 class="bar-mod-title">${_("Unit Location")}</h4>
                             <h5 class="title">${_("Location ID")}</h5>
                             <p class="unit-id">
                                 <span class="unit-id-value" id="unit-location-id-input">${unit.location.name}</span>
-                                <span class="tip"><span class="sr">Tip: </span>${_("Use this ID when you create links to this unit from other course content. You enter the ID in the URL field.")}</span>
+                                <span class="tip"><span class="sr">Tip: </span>${_('To create a link to this unit from an HTML component in this course, enter "/jump_to_id/{location ID}" as the URL value.')}</span>
```

Comment:
I'm concerned about the curly braces.  The translators have been taught to leave text in curly braces alone, but here they should translate it. I also don’t know if users will understand what the braces are doing there.